docs/apps/uaa/readme.md

```diff
@@ -0,0 +1,173 @@
+# Deploy UAA as a Kf App
+
+These instructions will walk you through deploying [Cloud Foundry UAA](github.com/cloudfoundry/uaa) as a Kf app.
+
+## Deploy
+
+1. Download and extract the UAA source:
+
+    ```sh
+    wget https://github.com/cloudfoundry/uaa/archive/4.35.0.zip
+    unzip 4.35.0.zip
+    cd uaa-*
+    ```
+
+1. Add a default servlet mapping for Tomcat (this is necessary as the
+   Tomcat Cloud Native Buildpack does not include a global web.xml with this
+   mapping.):
+
+    ```sh
+    cat >uaa/src/main/webapp/WEB-INF/tomcat-web.xml <<EOF
+    <?xml version="1.0" encoding="ISO-8859-1"?>
+    <web-app xmlns="http://java.sun.com/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" version="3.0"
+      metadata-complete="true">
+      <servlet>
+            <servlet-name>default</servlet-name>
+            <servlet-class>
+              org.apache.catalina.servlets.DefaultServlet
+            </servlet-class>
+            <init-param>
+                <param-name>debug</param-name>
+                <param-value>0</param-value>
+            </init-param>
+            <init-param>
+                <param-name>listings</param-name>
+                <param-value>false</param-value>
+            </init-param>
+            <load-on-startup>1</load-on-startup>
+        </servlet>
+    </web-app>
+    EOF
+    ```
+
+1. Build the UAA app:
+
+    ```sh
+    ./gradlew :cloudfoundry-identity-uaa:war
+    ```
+
+1. Create a sample deployment manifest. This snippet contains a functional UAA configuration. You should replace the embedded RSA keys and certs with your own:
```

Comment:
should we link to docs about how to do this?

Reply:
If those docs existed, yes.

Reply:
(Adding a link to the procedure I used to generate the sample manifest)